src/fs.cpp

```diff
@@ -6,53 +6,51 @@
 #include <fstream>
 #include <string_view>
 
-namespace filesystem {
-    Buffer file_read(std::string_view filename) {
-        const auto &filename_str = std::string(filename);
+Buffer file_read(std::string_view filename) {
```

Comment:
why remove the namespace? please revert this
rename `filesystem` to `fs` if you want it to be shorter